.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+name: Build
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Maven Cache"
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+
+      - name: "Setup Java"
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.2
+
+      - name: "Maven Tests"
+        run: |
+          mvn test-compile -DskipTests -Dmaven.javadoc.skip=true -B -V
+          mvn clean test jacoco:report coveralls:report
```

Comment:
This will need to be a dry run as to not submit coveralls report twice. Please see argument from other workflows.